### PR TITLE
Use Purescript v0.11.1

### PR DESCRIPTION
**Pull request by lpil**
This is the latest on npm

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -5,7 +5,7 @@ sudo: false
 install:
   - nvm install 6
   - nvm use 6
-  - npm install -g purescript@0.10.2 pulp bower
+  - npm install -g purescript@0.10.7 pulp@10.0.4 bower
```
Comment (paf31) [inline .travis.yml]: You can specify `0.11.1` and `11.0.0` here now. They are on NPM, but labelled `next`.

Comment (lpil) [inline .travis.yml]: That doesn't work for me.

They don't seem to be on npm yet- is this still the correct package? https://www.npmjs.com/package/purescript

Comment (lpil) [inline .travis.yml]: Ah, seems `0.11.1` was published to npm since I made this PR :)

**Comment by lpil** [comment]
I don't seem to be able to break the deps cache so I can't upgrade the prelude package to one that supports v0.11.

File: exercises/word-count/bower.json [hunk 2]
```diff
@@ -7,7 +7,7 @@
     "output"
   ],
   "dependencies": {
-    "purescript-prelude": "^2.5.0",
+    "purescript-prelude": "^3.0.0",
     "purescript-maps": "^2.1.2"
```
Comment (paf31) [inline exercises/word-count/bower.json]: You will need to update all of these to the latest versions, including `devDependencies`.

**Comment by lpil** [comment]
I was going to do that after I worked out how to update just one. This alone isn't enough to break the cache and cause the newer version to be downloaded it seems.

I've been given commit, so hopefully I can break the cache manually now.

**Comment by lpil** [comment]
Now that the cache is going compiling each exercise takes too long for the build to finish.

Could we share some of the compiled modules between exercises?

**Comment by paf31** [comment]
Yes, if we share and cache `output/`.

**Comment by lpil** [comment]
Much better, thanks.

How's that? :)

**Comment by paf31** [comment]
Great, thanks very much!

**Pull request merged**